src/test/java/org/jenkins/tools/test/PluginCompatTesterTest.java

```diff
@@ -87,176 +63,6 @@ public void setUp() throws Exception {
         }
     }
 
-    @Ignore("TODO broken by https://github.com/jenkinsci/active-directory-plugin/releases/tag/active-directory-2.17; figure out how to pin a version")
```

Comment:
As explained in the PR description, most of these tests were already disabled or testing unused modes. A new test suite is overdue but not handled in this PR.